core/include/moveit/task_constructor/stage_p.h

```diff
@@ -131,6 +132,12 @@ class StagePrivate
 	bool storeSolution(const SolutionBasePtr& solution);
 	void newSolution(const SolutionBasePtr& solution);
 	bool storeFailures() const { return introspection_ != nullptr; }
+	void runCompute() {
```

Comment:
@rhaschke would it acceptable to fold the code inside runCompute() inside compute()?